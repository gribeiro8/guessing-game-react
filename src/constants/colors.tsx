/**
 * Centralizei as cores do projeto
 */

export const COLORS = Object.freeze({
  transparent: 'rgba(0,0,0,0)',
  black: '#000000',
  black2: '#262A34',
  white: '#ffffff',
  red: '#CC3300',
  orange: '#FF6600',
  green: '#32BF00',
  grey1: '#CFCFCF',
  grey2: '#DDDDDD',
  grey3: '#9e9e9e',
});
