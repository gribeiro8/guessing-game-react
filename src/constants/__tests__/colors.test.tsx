import { COLORS } from '../colors';

describe('COLORS', () => {
  it('should have expected values', () => {
    expect(COLORS).toMatchSnapshot();
  });
});
