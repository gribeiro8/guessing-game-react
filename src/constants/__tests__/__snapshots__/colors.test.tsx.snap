// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COLORS should have expected values 1`] = `
Object {
  "black": "#000000",
  "black2": "#262A34",
  "green": "#32BF00",
  "grey1": "#CFCFCF",
  "grey2": "#DDDDDD",
  "grey3": "#9e9e9e",
  "orange": "#FF6600",
  "red": "#CC3300",
  "transparent": "rgba(0,0,0,0)",
  "white": "#ffffff",
}
`;
